cassiopeia/simulator/__init__.py

```diff
@@ -0,0 +1,3 @@
+from DataSimulator import DataSimulator
+from LineageTracingDataSimulator import LineageTracingDataSimulator
+from TreeSimulator import TreeSimulator
```

Comment:
what's the difference/benefit/caveat of using `from .DataSimulator import DataSimulator` vs `from DataSimulator import DataSimulator`?

Reply:
I'd want to believe there is no difference? If we've been using `from .X`  then let's use this for uniformity.